js/models/addressBook_model.js

```diff
@@ -2,12 +2,17 @@ angular.module('contactsApp')
 .factory('AddressBook', function()
 {
 	return function AddressBook(data) {
+		var storageKey = btoa(data.data.href.split('/').splice(-3, 2).join('_'));
 		angular.extend(this, {
 
 			displayName: '',
 			contacts: [],
 			groups: data.data.props.groups,
 			readOnly: data.data.props.readOnly === '1',
+			// use only owner and url, convert ''/remote.php/dav/addressbooks/users/admin/Test1/'' to 'admin.Test'
+			key: data.data.href.split('/').splice(-3, 2).join('_'),
+			enabled: window.localStorage.getItem('contacts_ab_'+this.key) !== false,
```

Comment:
this doesn't work, as localstorage only stores Strings.
=> this statement always evaluates to true.

Reply:
Indeed thx

Reply:
Hum, it's strange since it actually remembers the state, so It seems to work :)